build.gradle

```diff
@@ -16,17 +16,8 @@ checkstyleMain {
     source = 'src/main/java'
 }
 
-task runLangtonAntAdrian (dependsOn: 'classes', type: JavaExec) {
```

Comment:
The tasks for Adrian and Adhemar are being deleted, maybe this happened during conflicts resolution. Please restore those tasks (it can be a new commit) so the code from them is not lost

Same for the java files under their folders that were deleted

Reply:
I didn't realize that! Their files were restored as well their tasks.